packages/dialtone-tokens/build-sd-transforms.js

```diff
@@ -16,6 +15,18 @@ register(StyleDictionary);
 
 registerDialtoneTransforms(StyleDictionary);
 
+/**
+ * Transform a string from kebab-case to PascalCase
+ * @param string
+ * @returns {string}
+ */
+function kebabCaseToPascalCase (string) {
+  return string?.toLowerCase()
+    .split('-')
+    .map(word => word.charAt(0).toUpperCase() + word.slice(1))
+    .join('');
+}
+
```

Comment:
Why moving it to this file? I think that utility it's more general than just for build-sd-transforms, don't you think?

Reply:
Moved back with `.mjs` change mentioned above.